ng2-components/ng2-activiti-tasklist/src/components/activiti-people.component.spec.ts

```diff
@@ -148,36 +119,39 @@ describe('ActivitiPeople', () => {
         });
 
         it('should show people involved', () => {
-            expect(element.querySelector('#user-fake-id')).not.toBeNull();
```

Comment:
structure and layout of the component is changed. According to new changes there is no user-fake-id element.